frontend/src/components/audit-log/AuditLog.vue

```diff
@@ -25,46 +25,11 @@
 import AuditEntry from './AuditEntry.vue'
 import FFAccordion from '@/components/Accordion.vue'
 
-const eventDescriptions = {
-    'project.created': 'Project created',
-    'project.duplicated': 'Project duplicated',
-    'project.deleted': 'Project deleted',
-    'project.stopped': 'Project stopped',
-    'project.started': 'Project started',
-    'auth.login': '${user} logged in',
-    'auth.login.revoke': '${user} logged out',
-    'flows.set': 'Flows updated',
-
-    // Project Log
-    'user.invited': 'User invited',
-    'user.uninvited': 'User uninvited',
-    'user.invite.accept': '${user} accepted invitation',
-    'user.invite.reject': '${user} declined invitation'
-
-}
-
-const eventIcons = {
-    'project.started': 'play',
-    'project.stopped': 'stop',
-    'project.created': 'create',
-    'project.duplicated': 'create',
-    'project.deleted': 'create',
-    'auth.login': 'user',
-    'auth.login.revoke': 'logout',
-    'flows.set': 'pencil'
-}
-
```

Comment:
This should have all been removed with the last Audit Log work as we now use `AuditLogVerbose` to get descriptions